src/EditorFeatures/Core/ExternalAccess/VSTypeScript/Api/AbstractVSTypeScriptRequestHandlerProvider.cs

```diff
@@ -1,19 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Collections.Immutable;
-using Microsoft.CodeAnalysis.LanguageServer;
-using Microsoft.CodeAnalysis.LanguageServer.Handler;
-
-namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api;
-
-internal abstract class AbstractVSTypeScriptRequestHandlerProvider : IRequestHandlerProvider
```

Comment:
⚠️ This is problematic. Do we know if the VSTypeScript services are stateful?

Reply:
as of today, there aren't any (this was added recently) - there will be some coming and currently will be stateful